lib/go-tc/service_category.go

```diff
@@ -37,3 +39,25 @@ type ServiceCategory struct {
 	LastUpdated TimeNoMod `json:"lastUpdated" db:"last_updated"`
 	Name        string    `json:"name" db:"name"`
 }
+
+// ServiceCategoriesResponseV50 is a list of Service Categories as a response.
+type ServiceCategoriesResponseV50 struct {
+	Response []ServiceCategoryV50 `json:"response"`
+	Alerts
+}
+
+// ServiceCategoryResponseV50 is a single Service Category response for Update and Create to
```

Comment:
GoDoc comments should be a complete sentence; I think this was cut off before it was complete.

Reply:
Yes, It was cut off. Added it back.